governance/polls/Vault Compensation Poll 3

```diff
@@ -0,0 +1,54 @@
+
+title: Vault Compensation Poll
+summary: Signal your support for vault compensation
+discussion link: https://forum.makerdao.com/t/vault-compensation-plan-v2/3584
+vote type: Ranked Choice Voting
+options: 
+   0: Abstain
+   1: 24.67% (theoretical maximum auction yield) - up to 25,925 MKR and 50,000 DAI 
+   2: 21% - up to 21,627 MKR and 50,000 DAI
+   3: 18% - up to 18,216 MKR and 50,000 DAI
+   4: 15% - up to 14,871 MKR and 50,000 DAI
+   5: 12% - up to 11,691 MKR and 50,000 DAI
+   6: 9% - up to 5,592 MKR and 50,000 DAI
+   7: 6% - up to 2,748 MKR and 50,000 DAI
+   8: 3% - up to 917 MKR and 50,000 DAI
+   9: 0% (oppose vault compensation) - 0 MKR and 0 DAI
+   10: Oppose current compensation plan (support vault compensation) 0 MKR and 0 DAI
+---
+# Vault Compensation Plan Approval Poll
+
+The Maker Foundation Interim Governance Facilitator has placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of the community, which the community can use to signal support for compensating vaults affected by liquidations during the March 12 market crash.
```

Comment:
Interim! How very dare you. Also shouldn't be Maker Foundation. 'MakerDAO Governance Facilitator.' or 'Maker Governance Facilitator'

Reply:
Also, I feel like poll duration should be addressed here. 2 weeks seems good to me, but open to it being longer if necessary.

Reply:
Updated! And whooops, sorry about that :)